src/directives/badge.js

```diff
@@ -9,10 +9,18 @@ function directive (el, binding) {
       icon: false,
       left: false,
       bottom: false,
-      overlap: false
+      overlap: false,
+      visible: true
```

Comment:
I like this, but whatever property name is used, we must ensure that if we add functionality similar to other directives, or if there is already functionality like this, that the names match up. Similar API is important.